examples/structured_data/classification_with_grn_and_vsn.py

```diff
@@ -96,7 +96,7 @@
     "country_of_birth_self",
     "citizenship",
     "own_business_or_self_employed",
-    "fill_inc_questionnaire_for_veteran's_admin",
+    "fill_inc_questionnaire_for_veterans_admin",
```

Comment:
Have added this minor change because while saving the model was getting the following error:
https://github.com/tensorflow/tensorflow/issues/44984